src/lib/accessibility/interfaces.ts

```diff
@@ -178,4 +178,4 @@ export type ActionsKeyHandler = {
 export type KeyboardHandler = (event: KeyboardEvent) => void
 export type EventHandler = (event: Event) => void
 
-export type Accessibility = IAccessibilityDefinition | ((props: any) => IAccessibilityDefinition)
+export type Accessibility = ((props: any) => IAccessibilityDefinition)
```

Comment:
More important, any specific reason why we don't support just object definition, but only functions now? What is the problem of having both?

Reply:
We could have both object and functions, but as described above, it started to bring complexities which are more seen in the future, here are a couple of reasons:
- we always have to check whether we get function or object (as we aim to make accessibility framework agnostic, it can also bring some stress for the users, as they need to implement custom behavior derived from ```Accessibility``` type).
- If we want to reuse some parts of other behavior, it would be quite unsafely or even impossible to make it nicely
- Already seen a not good way of implementation in ```ListItemBehavior```, where we have to use a double function to pass props
- And since most of the behaviors (for now like 80-90%) needs props to correctly render aria*-attributes, I don't see a reason to support such ambiguous structure for the rest of 10%